tests/lca/imagebasedupgrade/cnf/upgrade-talm/tests/upgrade-e2e-test.go

```diff
@@ -49,44 +49,6 @@ var _ = Describe(
 			})
 		})
 
-		AfterAll(func() {
```

Comment:
If we just remove this, we are basically loosing test coverage for rollback. 

I suggest we do this: 
- Use Ordered container (without continue on error)
- TC1: upgrade
- TC2: rollback (use this test id for rollback: 69058) - use different test tag, so we can choose to not run this test for debugging purpose.
- Update polarion test case 68954 to remove the teardown to match the actual implementation.

Reply:
@yliu127  You are right that we would lose coverage for rollback after successful upgrade. We do cover rollback in the [rollback after failed upgrade test case](https://github.com/openshift-kni/eco-gotests/blob/main/tests/lca/imagebasedupgrade/cnf/upgrade-talm/tests/rollback-after-failed-upgrade-test.go). 

 The goal of this PR was to have the cluster in an upgraded state after the test suite. If we break this test into two test cases as you suggest we will still wind up with a rolled back cluster after the suite finishes. If you are okay with that, I'll make the changes you recommended.

Reply:
yes. but those are two different test cases  - rollback after successful upgrade vs auto rollback on failed upgrade.

Reply:
With the two test cases separated, we can optionally include/exclude rollback after successful upgrade.

Reply:
@josclark42 just to confirm, is polarion test case 68954  updated with teardown removed?

Reply:
@yliu127 I revised the test case by setting it to 'unapproved' first, as you suggested.